tests/NoLeftPizzaTest.elm

```diff
@@ -135,6 +135,52 @@ f =
      y)
         """
                         ]
+        , test "handle parser operators with pizza" <|
+            \() ->
+                """
+module MyParser exposing (..)
+numberToken =
+    Parser.getChompedString <|
+        Parser.succeed ()
+            |. Parser.chompIf Char.isDigit
+            |. Parser.chompWhile Char.isDigit
+"""
+                    |> Review.Test.run NoLeftPizza.rule
+                    |> Review.Test.expectErrors
+                        [ makeError """Parser.getChompedString <|
+        Parser.succeed ()
+            |. Parser.chompIf Char.isDigit
+            |. Parser.chompWhile Char.isDigit"""
+                            |> Review.Test.whenFixed
+                                """
+module MyParser exposing (..)
+numberToken =
+    Parser.getChompedString (Parser.succeed () |. Parser.chompIf Char.isDigit |. Parser.chompWhile Char.isDigit)
+"""
+                        ]
+        , test "ignores logic operators" <|
+            \() ->
+                """
+module A exposing (..)
+f =
+    if isTrue <| True || False then
+        True
+    else
+        False
+"""
+                    |> Review.Test.run NoLeftPizza.rule
+                    |> Review.Test.expectErrors
+                        [ makeError "isTrue <| True"
```

Comment:
Is `isTrue <| True || False` equivalent to `(isTrue True) || False`? I thought it was equivalent to `isTrue <| (True || False)`, which if I'm right makes the fix change the behaviour of the expression.

Reply:
Ah, I was assuming "left" vs "right" is the way to go... do we need to go and find the operator weight and compare that? 😟

Reply:
Well we shouldn't have to, as it is `elm-syntax`'s job, but it currently doesn't do it.(https://github.com/stil4m/elm-syntax/issues/41
https://github.com/stil4m/elm-syntax/issues/37)

Reply:
Ah, in that case I've misunderstood the purpose of `dir` and this probably needs a whole bunch more tests, and possibly an operator table, to get this right.

Reply:
Update on that thought, since `<|` and `|>` are at the top of the operator table then everything else should be grabbed. I'll push that up and then consider what to do when we encounter `|>`.

Reply:
@jfmengels confirmed for me that `a <| b |> c` won't compile with the error "I do not know how to group these expressions. Add parentheses for me!" at which case it's just two expressions either side of a pizza.